src/ocean/text/arguments/DateIntervalArgs.d

```diff
@@ -0,0 +1,429 @@
+/*******************************************************************************
+
+    Handler for date interval CLI arguments
+
+    copyright: Copyright (c) 2018 dunnhumby Germany GmbH. All rights reserved
+
+*******************************************************************************/
+
+module ocean.text.arguments.DateIntervalArgs;
+
+import core.stdc.time;
+
+import ocean.util.app.model.IApplication;
+import ocean.text.Arguments;
+import ocean.text.convert.DateTime;
+import ocean.text.convert.Integer;
+import ocean.transition;
+
+version (UnitTest)
+{
+    import Test = ocean.core.Test;
+}
+
+/// Class that handles the begin-date and end-date CLI arguments
+public class DateIntervalArgs
+{
+    /// The begin and end of the run, if given on the command line
+    private long cli_begin;
+
+    /// ditto
+    private long cli_end;
+
+    /// The interval that should be used when dates are not set by the user
+    private long default_interval;
```

Comment:
Default interval in what? Years? Seconds? Minutes? I assume seconds so i'd rename to default_interval_sec.